go/logic/migrator.go

```diff
@@ -1279,30 +1279,31 @@ func (this *Migrator) executeWriteFuncs() error {
 					return err
 				}
 			}
-		default:
-			{
+		case copyRowsFunc := <-this.copyRowsQueue:
+		priority:
```

Comment:
Would moving [line 1283 to 1295](https://github.com/github/gh-ost/pull/1124/files#diff-bd6a81585ccffb0f0feef9d5b3d1c542432f90c29a366a7bfb1e96531d36cbbaR1283-R1295) to a new function such as `migrator.drainApplierEventQueue()` remove the need for `priority:` label + `break priority`?

I wonder if this is easier to read (if I understand correctly)

Reply:
yeah, your suggestion is right